src/app/static/src/js/components/runReport/reportList.vue

```diff
@@ -4,8 +4,9 @@
         :serializer="e => e.name"
         v-model="query"
         showOnFocus
+        maxMatches="50"
         placeholder="Choose a report"
-        @hit="$emit('update:report', $event.name)"
+        @hit="$emit('update:report', $event.name); $emit('update:key', $event.key)"
```

Comment:
I am not sure why you need to emit report key here.

Reply:
Yep, not immediately clear to me of difference between `key` and `report` (or between `selectedLogReportKey` and `selectedReport`)

Reply:
Yeah, this line doesn't seem relevant to this ticket. This is just where you select the report that you want to run, it's nothing to do with selecting a report that is already running.